cmd/soroban-cli/src/contract/deploy.rs

```diff
@@ -106,6 +108,9 @@ impl Cmd {
             self.wasm_hash.as_ref().unwrap().to_string()
         };
 
+        // Add sleep so horizon can ingest new account state
+        sleep(Duration::from_secs(2)).await;
```

Comment:
Can you add a comment indicating this should be removed once `getAccount` is removed?